### ISSUE TITLE
Dynamic currency

### ISSUE THREAD

**Issue by connected** [kazupon/vue-i18n#242]
I have the following config for currency:
```js
new VueI18n({
  locale: 'en-GB',
  numberFormats: {
    'en-GB': {
      currency: {
        style: 'currency',
        currency: 'gbp'
      }
    },
    'ru-RU': {
      currency: {
        style: 'currency',
        currency: 'rub'
      }
    },
    'lv-LV': {
      currency: {
        style: 'currency',
        currency: 'eur'
      }
    }
  }
})
```
In my templates I have: 
```
{{ $n(100, 'currency') }}
```
that results in:
```
 £100.00
```
This is ok for UK user, but if I am from another country (lets say Latvia) and I set `i18n.locale = 'lv-LV'`
I get euro currency: 
```
100,00 €
```
instead of pounds
```
100,00 £
```

When I use raw Intl.NumberFormat, I can pass currency as parameter like so
```js
Intl.NumberFormat("lv-LV", { style: 'currency', currency:"gbp", currencyDisplay: "symbol"}).format(100);
Intl.NumberFormat("en-GB", { style: 'currency', currency:"gbp", currencyDisplay: "symbol"}).format(100);
```

How do I achieve the same with `$n`, am I missing something here?

**Comment by kazupon** [comment]
Close (with #305 )

**Comment by PrimozRome** [comment]
@kazupon I need the same functionality to set currency code dynamically as @connected described above. I see this was implemented with #305, but I can not find in documentation how to use it?
